src/arrayMethodSort.js

```diff
@@ -4,8 +4,17 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = (prev, curr) =>
+    (String(prev) > String(curr))) {
```

Comment:
It's better to create a separate function and use it here as a default value for a compareFunction